src/config.py

```diff
@@ -13,3 +13,7 @@
 load_dotenv()
 
 GOOGLE_PLACES_API_KEY = os.getenv("GOOGLE_PLACES_API_KEY")
+
+FACEBOOK_SDK_GRAPH_API_KEY = os.getenv("FACEBOOK_SDK_GRAPH_API_KEY")
```

Comment:
You can add the new needed values to the .emv.template

Reply:
Addressed.